docs/pages/includes/database-access/reference/aws-iam/documentdb/role-as-user-policy.mdx

```diff
@@ -0,0 +1,6 @@
+The Teleport Database Service assumes a IAM role when connecting to a
+DocumentDB cluster with IAM authentication.
+
+Refer to the [Authentication using IAM
```

Comment:
```suggestion
Refer to [Authentication using IAM
```

"The" doesn't refer to anything here. An alternative would be to keep "the" but say, "The Authentication using IAM Identity _guide_", so "the" would modify "guide".